client/style.css

```diff
@@ -0,0 +1,14 @@
+.kanban-board {
```

Comment:
css should be genareted during gulp task (all css styles should be in *.scss files)